### DISCUSSION TITLE
Download zip from repo not working

### DISCUSSION THREAD

**Discussion by bianchui** [community/community#12497] [Other Feature Feedback, Questions, & Ideas]
Download zip from repo not working
To reproduce:

1. find any repo like https://github.com/github/docs
2. select Code -> Download Zip
3. get 404 error

**Comment by Lanius-collaris** [comment]
Reproducible with:
- Windows 7

Not reproducible with:
- Android 10
- Linux Mint
- Alpine Linux

 Is it related to DNS?
